package.json

```diff
@@ -75,9 +75,7 @@
     "qrcode": "1.2.0",
     "raven": "2.1.2",
     "redis": "2.7.1",
-    "request": "2.79.0",
-    "restify": "4.3.0",
```

Comment:
@vladikoff I removed restify as a dependency, however the [README](https://github.com/mozilla/fxa-auth-server/blob/0c0ef301ac9ee204dc94ba87f2051fd55bf9636d/README.md#mailer), still states that it is used? I didn't find any references to restify in the code, but could I be missing something here?

Reply:
Seems like old mailer info? we should remove it